pkg/metrics/producers/scheduledcapacity/producer.go

```diff
@@ -25,5 +28,67 @@ type Producer struct {
 
 // Reconcile of the metrics
 func (p *Producer) Reconcile() error {
+	var (
+		loc     *time.Location
+		err     error
+		currVal int32
+	)
+	matchFound := false
+	now := time.Now()
+
+	if p.Spec.Schedule.Timezone != nil {
+		loc, err = time.LoadLocation(*p.Spec.Schedule.Timezone)
+		if err != nil {
+			return fmt.Errorf("timezone was not a valid input")
+		}
+	} else {
+		// Set Default location to UTC if not specified
+		loc = time.UTC
+	}
+	now = now.In(loc)
+
+F:
+	for _, behavior := range p.Spec.Schedule.Behaviors {
+		// Turn Strongly-typed into crontab
+		crontabStart := parseScheduleSpecIntoCrontab(behavior.Start)
```

Comment:
I might name this something like crontabFrom(...) which implies all of the other language bits around parsing into

Reply:
Sounds good to me.